src/components/CounterValue/index.js

```diff
@@ -45,9 +49,10 @@ const mapStateToProps = (state: State, props: OwnProps) => {
   const counterValueCurrency = counterValueCurrencySelector(state)
   const counterValue =
     !counterValueCurrency || !currency
```

Comment:
none of these case should happen now, (currency mandatory / countervalueCurrency always available)